### ISSUE TITLE
Does anyone have it running properly?

### ISSUE THREAD

**Issue by kfa1983** [endtwist/AjaxIM#50]
Hey guys,

I've been trying to make it work for like 3 days now and I can't think of anything to fix it... And when I fix something, I get a new error...

Newest error:

Error: {"type":"error","error":"not authenticated"}
    at Object.session [as handle](/root/ajaxim-latest/server/middleware/im.js:62:18)
    at next (/root/node_modules/express/node_modules/connect/lib/http.js:204:15)
    at Object.bodyParser [as handle](/root/node_modules/express/node_modules/connect/lib/middleware/bodyParser.js:59:61)
    at next (/root/node_modules/express/node_modules/connect/lib/http.js:204:15)
    at Object.cookieParser [as handle](/root/node_modules/express/node_modules/connect/lib/middleware/cookieParser.js:44:5)
    at next (/root/node_modules/express/node_modules/connect/lib/http.js:204:15)
    at Object.methodOverride [as handle](/root/node_modules/express/node_modules/connect/lib/middleware/methodOverride.js:35:5)
    at next (/root/node_modules/express/node_modules/connect/lib/http.js:204:15)
    at Object.handle (/root/node_modules/express/lib/http.js:85:5)
    at next (/root/node_modules/express/node_modules/connect/lib/http.js:204:15)

But on http://localhost:8000/dev/Cookie I get the "Cookie set" message, what is this happening? 

On http://localhost:8000/dev, I just get the "hello" message and nothing else, no bar, no nothing... that's pretty exhausting too plus I don't even know if it'll work at the end..

H.E.L.P, please...

Thank you

**Comment by sharjeel** [comment]
I'm also trying to make it run and not seeing any chatbar. 

Firstly, the jade files seem to be problematic and hence generating incorrect HTML. 

I fixed HTML issues by changing colons to equals in chat.jade. i.e. " src= '/js/.. " instead of " src: '/js/.. " and similarly " type= 'text/javascript' " instead of " type: 'text/javascript' ". 

This gave correct HTML rendering (view source in browser confirms that) but the Javscript and other static links do not load due to authentication middleware malfunctioning: gives 500 error responses. 

I tried disabling auth middleware and the static content started serving well. 

Anybody knows whats wrong with auth middeware?
